Makefile

```diff
@@ -61,7 +65,7 @@ currentshell: .env .docker-build
 
 .PHONY: psql
 psql: .env .docker-build
-	docker-compose run db psql -h db -U postgres
+	docker-compose run db psql --host db --username postgres
```

Comment:
Using the long flags makes this easier to read. I can never remember what `-U` is.